client/blocks/login/login-form.jsx

```diff
@@ -430,7 +436,7 @@ export class LoginForm extends Component {
 
 						<FormLabel htmlFor="usernameOrEmail">
 							{ this.isPasswordView() ? (
-								<Button
+								<A8CButton
```

Comment:
Given that `Button` from `@automattic/components` is deprecated, can this be an opportunity to switch to the `Button` component from `@wordpress/components` ?

Reply:
This is a different Woo form to the one I've seen in production, and as such I haven't touched it at all. I've intentionally tried to keep to a limited scope but I'm aware this has likely created inconsistencies.

- [ ] I'll find out how to test this flow.

Reply:
This turns out be [very intensive to test](https://github.com/Automattic/wp-calypso/pull/41798) and I have not managed to do so, but I can force this state locally and the button replacements are straightforward so I've [committed](https://github.com/Automattic/wp-calypso/pull/103616/commits/00733942c633d9690c43e417a2f16221904a96b8) them.